### DISCUSSION TITLE
Exciting repo!

### DISCUSSION THREAD

**Discussion by trevorgildersleeve** [awslabs/sbt-aws#9] [General]
I just wanted to drop a comment here that I'm very excited to see this repo / project! 

I'm in the early stages of designing my first SaaS application, but have been learning CDK and using Amplify (Gen 1 & 2 preview) for basic mock-ups.

 This project seems to be in it's very early stages? Are there any AWS articles mentioning this repo / initiative?

**Comment by tobuck-aws** [comment]
@trevorgildersleeve, thanks! Yes, this project is very early stages. We'll soon publish a roadmap project to give you more insight into where we're going and some sense of timelines. Most of the documentation for this project is in the project at this point. We haven't quite launched (so thanks for finding us in spite of that!), but once we do we'll have more collateral to point you to.

In the interim, if you have questions, feel free to post here. Happy to try and help

**Reply by trevorgildersleeve** [reply]
Thanks! I'm really looking forward to what'll come out on this. While Amplify Gen 2 is amazing, I feel like this will be a better starting point in the long run. Although, if possible, spinning all of this up AS an Amplify Gen 2 project would be off the charts!

**Reply by tobuck-aws** [reply]
@trevorgildersleeve we're starting to look into Amplify Gen 2 integration. Can you provide any details into what, specifically, you'd like to see? Are you hoping for components in the app-plane side of things that help you to bootstrap a multi-tenant web app? Something on the control plane side, maybe the admin console? Or something altogether different? Would love to hear your thoughts!
